### PR TITLE
Add gcloud image!

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related: 

### Pre-review Checklist

- [X] **IMPORTANT: 'image-request' tag has been applied if this PR is adding any images, including new versions or variants**

#### For new image PRs only

If you have an apko.yaml file in this PR you need to follow this checklist, otherwise feel free to remove.
- [X] Image is marked experimental or stable as appropriate
- [ ] The last two minor versions are available
- [X] The latest tag points to the newest stable version
- [ ] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [X] The image runs as `nonroot` and GID/UID are set to 65532
  - [X] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
  - [ ] See above for exceptions to nonroot rule
- [X] ENTRYPOINT
  - [X] For applications/servers/utilities call main program with no arguments e.g. [redis-server]
  - [X] For base images leave empty
  - [X] For dev variants set to entrypoint script that falls back to system
- [X] CMD:
  - [X] For server applications give arguments to start in daemon mode (may be empty)
  - [X] For utilities/tooling bring up help e.g. `–help`
  - [X] For base images with a shell, call it e.g. [/bin/sh]
- [X] Consider where and how the image deviates from popular alternatives. Is there a good reason and is it documented?
- [X] Add annotations e.g:
```
annotations:
  "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
  "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/ # use the academy site here
  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/bazel # use github here
```
- [X] Check if environment variables are needed e.g. to set data locations
- [X] Ensure the image responds to SIGTERM
  - [X] `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`
- [X] Documentation. Let's make this excellent. Include usage example.
- [X] Error logs write to stderr and normal logs to stdout. DO NOT write to file.
- [X] Include tests, at the very least a basic smoke test.

### CODE REVIEW

File: images/google-cloud-sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/google-cloud-sdk:latest
+```
+
+## Using Vault
+
+The Chainguard Vault image contains the `google-cloud-sdk` tools, including `gcloud`, `gsutil`, `bq`, and more.
```
Comment (rawlingsj) [inline images/google-cloud-sdk/README.md:26]: looks great! just noticed this copy / paste

Comment (rawlingsj) [reply images/google-cloud-sdk/README.md:26]: and the heading above

File: images/google-cloud-sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+```
+
+## Using Google Cloud SDK
+
+The Chainguard Google Cloud SDK image contains the `google-cloud-sdk` tools, including `gcloud`, `gsutil`, `bq`, and more. 
+
+The default command is `gcloud`.
+
+```shell
+$  docker run  cgr.dev/chainguard/google-cloud-sdk:latest-arm64
```
Comment (jdolitsky) [inline images/google-cloud-sdk/README.md:31]:
```suggestion
$  docker run  cgr.dev/chainguard/google-cloud-sdk:latest
```

File: images/google-cloud-sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+  Anthos CLI
+      anthos                  Anthos command Group.
+
+```
+
+You can also use `gsutil` or `bq`:
+
+```shell
+$ docker run  cgr.dev/chainguard/google-cloud-sdk:latest-arm64 gsutil
```
Comment (jdolitsky) [inline images/google-cloud-sdk/README.md:59]:
```suggestion
$ docker run  cgr.dev/chainguard/google-cloud-sdk:latest gsutil
```

File: images/google-cloud-sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  hmac             CRUD operations on service account HMAC keys.
+  iam              Get, set, or change bucket and/or object IAM permissions.
+  kms              Configure Cloud KMS encryption
+  label            Get, set, or change the label configuration of a bucket.
+  lifecycle        Get or set lifecycle configuration for a bucket
+  logging          Configure or retrieve logging on buckets
+  ```
+
+  ```shell
+$ docker run  cgr.dev/chainguard/google-cloud-sdk bq
```
Comment (jdolitsky) [inline images/google-cloud-sdk/README.md:84]:
```suggestion
$ docker run  cgr.dev/chainguard/google-cloud-sdk:latest bq
```

**Comment by dlorenc** [comment]
Thanks!

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**